etc/config-template.toml

```diff
@@ -39,6 +39,15 @@
 ## It won't take effect when compiling without BCC_IOSNOOP=1. 
 # enable-io-snoop = true
 
+## Allow core dump being generated when TiKV panics. It is default to off to prevent
+## core dump from filling up disk space, especially when TiKV fall in crash loop.
+# allow-core-dump = false
```

Comment:
How about `abort-on-panic`?

Reply:
What are the other difference between abort and exit? I think core dump is more descriptive to what users can achieve with this config.

Reply:
Actually TiKV can't control whether to create a coredump, it's determined by several factors, all TiKV can do is just sending an abort signal.

Reply:
ok, will change the name.